cluster/saltbase/salt/kubelet/default

```diff
@@ -50,8 +50,8 @@
 {% set config = "--config=/etc/kubernetes/manifests" -%}
 
 {% set manifest_url = "" -%}
-{% if grains['roles'][0] == 'kubernetes-master' and grains.cloud in ['gce'] -%}
-  {% set manifest_url = "--manifest-url=http://metadata.google.internal/computeMetadata/v1/instance/attributes/google-container-manifest --manifest-url-header=Metadata-Flavor:Google" -%}
```

Comment:
We also need to be able to set the manifest-url-header, or else the metadata service will reject the kubelet's requests.

How about just making the pillar parameters the same as the apiserver flags? manifest_url and manifest_url_header